ftrr/src/main/java/framgia/co/edu/ftrr/util/ExcelExportUtils.java

```diff
@@ -2,91 +2,89 @@
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.util.Iterator;
+import java.io.*;
+import java.util.Date;
+import java.util.List;
 import java.util.Map;
 
 @Component
 public class ExcelExportUtils {
-    
-    private static final String TEMPLATE_FILE = "ExcelsTemplate/DivStatisticTemplate.xlsx";
+
+    private static final String TEMPLATE_FILE = "ExcelsTemplate/ExportTemplate.xlsx";
+    private static final Integer divisionColum = 0;
+    private static final Integer requestTimeColum = 1;
+    private static final Integer officeColum = 2;
+    private static final Integer skillColum = 3;
+    private static final Integer quantityColum = 4;
+    private static final Integer statusColum = 5;
+    private static final Integer resultColum = 6;
+    private static final Integer interviewColum = 7;
+    private static final Integer traineeJoinDivColum = 8;
 
     @Autowired
     DashboardService dashboardService;
 
-    public XSSFCellStyle createStyleForTitle(XSSFWorkbook workbook) {
-        XSSFFont font = workbook.createFont();
-        font.setBold(true);
-        XSSFCellStyle style = workbook.createCellStyle();
-        style.setFont(font);
-        return style;
+    public ByteArrayInputStream exportExcel(Date fromDate) {
+        try {
+            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            XSSFWorkbook xssfWorkbook = readTemplate();
+            processExcel(xssfWorkbook, dashboardService.getStatisticDataFollowDiv(fromDate));
+            xssfWorkbook.write(out);
+            return new ByteArrayInputStream(out.toByteArray());
+        } catch (IOException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    private void processExcel(XSSFWorkbook xssfWorkbook, Map<String, List<DivStatistic>> data) {
+        XSSFSheet rows = xssfWorkbook.getSheetAt(0);
+        Integer index = 1;
+        for (Map.Entry<String, List<DivStatistic>> entry : data.entrySet()) {
+            if (entry.getValue().size() == 0) continue;
+
+            Row row = rows.createRow(index);
+            row.createCell(divisionColum, CellType.STRING).setCellValue(entry.getKey());
+            index++;
+
+            List<DivStatistic> divStatistics = entry.getValue();
+
+            for (DivStatistic divStatistic : divStatistics) {
+                row = rows.createRow(index);
+                row.createCell(requestTimeColum).setCellValue(divStatistic.getRequestTime().toString());
+                row.createCell(skillColum).setCellValue(divStatistic.getSkill());
+                row.createCell(quantityColum).setCellValue(divStatistic.getQuantity());
+                row.createCell(statusColum).setCellValue(divStatistic.getStatus());
+                row.createCell(resultColum).setCellValue(divStatistic.getResult());
+                row.createCell(interviewColum).setCellValue(divStatistic.getInterview());
+                index++;
```

Comment:
add trainee join div rồi mới tăng index chứ hì

mà em check xem nếu status của div là hoàn thành rồi thì mới chạy cái in tên ứng viên join div cho khỏe, ko thì gọi code `continue` hì